### PR TITLE
Add entry: Changing how induction tutors can add themselves as a mentor

### PR DESCRIPTION
**Pull request by frankieroberto**
This design history shows how we removed the easy option for SITs to add themselves as a mentor, replacing it with an interstitial page, and the rationale for doing so.

### CODE REVIEW

File: app/posts/manage-training/2023-09-27-changing-how-induction-tutors-add-themselves-as-a-mentor.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+![Screenshot with the title “Are you sure you want to add yourself as a mentor?”](are-you-sure-you-want-to-add-yourself-as-a-mentor.png)
+
+
+If the induction tutor tries to add themselves as an early career teacher, we prevent this from happening, as this is strictly against the policy framework. Instead, we check whether they had meant to add themselves as a mentor, or would prefer to cancel.
+
+![Screenshot with the title “You cannot add yourself as an ECT”](you-cannot-add-yourself-as-an-ect.png)
+
+## Next steps
+
+We will monitor this impact this has on school induction tutors adding themselves as a mentor.
```
Comment (frankieroberto) [inline app/posts/manage-training/2023-09-27-changing-how-induction-tutors-add-themselves-as-a-mentor.md:48]:
```suggestion
We will monitor to see what impact this has on school induction tutors adding themselves as a mentor.
```

File: app/posts/manage-training/2023-09-27-changing-how-induction-tutors-add-themselves-as-a-mentor.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## What we changed
+
+We removed the option for induction tutors to add themselves as a mentor:
+
+![Screenshot with the title “Who do you want to add?” with radio button options for just 'ECT' and 'Mentor'](updated-who-do-you-want-to-add.png "The new “Who do you want to add?” screen")
+
+However, induction tutors can still add themselves as a mentor by using the “Add mentor” flow, and entering their own details.
+
+When this happens, we now present them with an interstitial screen advising them that the two roles are separate. The can then choose whether or not to continue.
```
Comment (Lloyd-K) [inline app/posts/manage-training/2023-09-27-changing-how-induction-tutors-add-themselves-as-a-mentor.md:37]: The**y** can then ...

File: app/posts/manage-training/2023-09-27-changing-how-induction-tutors-add-themselves-as-a-mentor.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+![Screenshot with the title “Are you sure you want to add yourself as a mentor?”](are-you-sure-you-want-to-add-yourself-as-a-mentor.png)
+
+
+If the induction tutor tries to add themselves as an early career teacher, we prevent this from happening, as this is strictly against the policy framework. Instead, we check whether they had meant to add themselves as a mentor, or would prefer to cancel.
```
Comment (Lloyd-K) [inline app/posts/manage-training/2023-09-27-changing-how-induction-tutors-add-themselves-as-a-mentor.md:42]: Change 'themselves' to 'themself' as the text refers to a single SIT. Thanks

File: app/posts/manage-training/2023-09-27-changing-how-induction-tutors-add-themselves-as-a-mentor.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+![Screenshot with the title “You cannot add yourself as an ECT”](you-cannot-add-yourself-as-an-ect.png)
+
+## Next steps
+
+We will monitor this impact this has on school induction tutors adding themselves as a mentor.
```
Comment (Lloyd-K) [inline app/posts/manage-training/2023-09-27-changing-how-induction-tutors-add-themselves-as-a-mentor.md]: We'll monitor **the** impact this has ...

**Review by Lloyd-K — CHANGES_REQUESTED** [review]

File: app/posts/manage-training/2023-09-27-changing-how-induction-tutors-add-themselves-as-a-mentor.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+## What we changed
+
+We removed the option for induction tutors to add themselves as a mentor:
+
+![Screenshot with the title “Who do you want to add?” with radio button options for just 'ECT' and 'Mentor'](updated-who-do-you-want-to-add.png "The new “Who do you want to add?” screen")
+
+However, induction tutors can still add themselves as a mentor by using the “Add mentor” flow, and entering their own details.
+
+When this happens, we now present them with an interstitial screen advising them that the two roles are separate. The can then choose whether or not to continue.
```
Comment (frankieroberto) [inline app/posts/manage-training/2023-09-27-changing-how-induction-tutors-add-themselves-as-a-mentor.md:37]:
```suggestion
When this happens, we now present them with an interstitial screen advising them that the two roles are separate. They can then choose whether or not to continue.
```

File: app/posts/manage-training/2023-09-27-changing-how-induction-tutors-add-themselves-as-a-mentor.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+![Screenshot with the title “Who do you want to add?” with radio button options for just 'ECT' and 'Mentor'](updated-who-do-you-want-to-add.png "The new “Who do you want to add?” screen")
+
+However, induction tutors can still add themselves as a mentor by using the “Add mentor” flow, and entering their own details.
+
+When this happens, we now present them with an interstitial screen advising them that the two roles are separate. They can then choose whether or not to continue.
+
+![Screenshot with the title “Are you sure you want to add yourself as a mentor?”](are-you-sure-you-want-to-add-yourself-as-a-mentor.png)
+
+
+If the induction tutor tries to add themselves as an early career teacher, we prevent this from happening, as this is strictly against the policy framework. Instead, we check whether they had meant to add themselves as a mentor, or would prefer to cancel.
```
Comment (frankieroberto) [inline app/posts/manage-training/2023-09-27-changing-how-induction-tutors-add-themselves-as-a-mentor.md:42]:
```suggestion
If the induction tutor tries to add themselves as an early career teacher, we prevent this from happening, as this is strictly against the policy framework. Instead, we check whether they had meant to add themself as a mentor, or would prefer to cancel.
```

File: app/posts/manage-training/2023-09-27-changing-how-induction-tutors-add-themselves-as-a-mentor.md (hunk 2 above)
Comment (frankieroberto) [reply app/posts/manage-training/2023-09-27-changing-how-induction-tutors-add-themselves-as-a-mentor.md]: Opps, thanks.

**Pull request merged**